js/src/components/google-combo-account-card/connect-ads/connect-ads-body.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import AdsAccountSelectControl from '.~/components/ads-account-select-control';
+import ContentButtonLayout from '.~/components/content-button-layout';
+import ConnectCTA from './connect-cta';
+import LoadingLabel from '.~/components/loading-label/loading-label';
+
+/**
+ * Clicking on the button to connect an existing Google Ads account.
+ *
+ * @event gla_ads_account_connect_button_click
+ * @property {number} id The account ID to be connected.
+ * @property {string} [context] Indicates the place where the button is located.
+ * @property {string} [step] Indicates the step in the onboarding process.
+ */
+
+/**
+ * ConnectAdsBody component.
+ *
+ * @param {Object} props Props.
+ * @param {Array} props.accounts Accounts.
+ * @param {Object} props.googleAdsAccount Google Ads account object.
+ * @param {boolean} props.isConnected Whether the account is connected.
+ * @param {Function} props.handleConnectClick Callback to handle the connect click.
+ * @param {boolean} props.isLoading Whether the card is in a loading state.
+ * @param {Function} props.setValue Callback to set the value.
+ * @param {string} props.value Ads account ID.
+ * @fires gla_ads_account_connect_button_click when "Connect" button is clicked.
+ * @return {JSX.Element} Body component.
+ */
+const ConnectAdsBody = ( {
+	accounts,
+	googleAdsAccount,
+	isConnected,
+	handleConnectClick,
+	isLoading,
+	setValue,
+	value,
+} ) => {
+	const connectedAccount = accounts.filter( ( account ) => {
+		return isConnected && account.id === value;
+	} );
+
+	return (
+		<ContentButtonLayout>
+			<AdsAccountSelectControl
+				accounts={ isConnected ? connectedAccount : accounts }
+				value={ value }
+				onChange={ setValue }
+				autoSelectFirstOption={ true }
```

Comment:
@ankitrox If there's a connected account, we can disable the select as such:
https://github.com/woocommerce/google-listings-and-ads/pull/2639/files#diff-c71501f6145dde774344794f6079c4d6c2f7199047c34fd0eba5484b220bde42R59

Reply:
Added it. Also, added these props in constant file so that we can use at both the places and will have a single source as object.